### ISSUE TITLE
using LvlFilterHandler in a library

### ISSUE THREAD

**Issue by tve** [inconshreveable/log15#63]
I'd like to create a context handler in or for a library (or sub-module) that filters out debug messages. Something like:
```
func NewFoo() *Foo {
    // create context logger
    foo := &Foo{ Logger: Log.New("obj", "foo") }
    // restrict context logger to Info and higher
    foo.SetHandler(log15.LvlFilterHandler(log15.LvlInfo, ???))
    return foo
}
```
the problem I'm running into is what to pass for the `???`. The library would have to be told about the next outer handler. If I do this outside of the library I still have the problem that a I need some global variable floating around to keep track of the root handler, and this defeats nesting. It would be nice to have a InsertHandler() that inserts a handler in front of the existing one. Or am I over-complicating?

**Comment by inconshreveable** [comment]
You're not over complicating. I think you're absolutely right that there should probably be an API for `GetHandler` that returns the Logger's current handler.

**Comment by ChrisHines** [comment]
Was this issue resolved by #75?

**Comment by inconshreveable** [comment]
Yep!
